common/src/main/scala/explore/common/package.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore
+
+package object common {
+  // These are needed for Reusability derivation macro to work when defined within traits.
```

Comment:
Will these be needed? If so, a better comment might be useful.

Reply:
These aren't needed either, and are leftovers from different things I tried.